package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "astro-typedoc",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.0",
```

Let’s keep `0.0.0` so in `Release 0.1` commit we will be able to increase the number.